core/src/main/java/io/grpc/internal/TransportTracer.java

```diff
@@ -0,0 +1,212 @@
+/*
+ * Copyright 2017, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.internal;
+
+import io.grpc.ExperimentalApi;
+import io.grpc.Status;
+import io.grpc.StreamTracer;
+import java.util.concurrent.Callable;
+import java.util.concurrent.atomic.AtomicLong;
+import javax.annotation.concurrent.ThreadSafe;
+
+/**
+ * A class for gathering statistics about a transport. This is an experimental feature.
+ */
+@ThreadSafe
+@ExperimentalApi
```

Comment:
This is internal, no need to label experimental.

Reply:
Done